cypress/components/dialog-full-screen/dialog-full-screen.cy.tsx

```diff
@@ -293,94 +287,83 @@ context("Testing DialogFullScreen component", () => {
         .should("have.css", "padding-right", "0px")
         .should("have.css", "padding-bottom", "0px");
     });
+
+    // skip this test for now as FE-6053 has not been fixed yet
+    it.skip("should render DialogFullScreen component with DisableClose prop", () => {
+      CypressMountWithProviders(<DialogFullScreenComponent disableClose />);
+      closeIconButton().click();
+      dialogFullScreenPreview().should("exist");
+    });
   });
 
   describe("should check accessibility for Dialog Full Screen", () => {
     it("should check accessibility for default Dialog Full Screen component", () => {
       CypressMountWithProviders(<DefaultDocsStory />);
 
-      buttonDataComponent()
-        .contains("Open DialogFullScreen")
-        .click()
-        .then(() => cy.checkAccessibility());
+      buttonDataComponent().contains("Open DialogFullScreen").click();
+      cy.checkAccessibility();
     });
 
     it("should check accessibility for default Dialog Full Screen with complex example", () => {
       CypressMountWithProviders(<WithComplexExample />);
 
-      buttonDataComponent()
-        .contains("Open DialogFullScreen")
-        .click()
-        .then(() => cy.checkAccessibility());
+      buttonDataComponent().contains("Open DialogFullScreen").click();
+      cy.checkAccessibility();
     });
 
     it("should check accessibility for default Dialog Full Screen with disabled content padding", () => {
       CypressMountWithProviders(<WithDisableContentPadding />);
 
-      buttonDataComponent()
-        .contains("Open DialogFullScreen")
-        .click()
-        .then(() => cy.checkAccessibility());
+      buttonDataComponent().contains("Open DialogFullScreen").click();
+      cy.checkAccessibility();
     });
 
     it("should check accessibility for default Dialog Full Screen component with header children", () => {
       CypressMountWithProviders(<WithHeaderChildren />);
 
-      buttonDataComponent()
-        .contains("Open DialogFullScreen")
-        .click()
-        .then(() => cy.checkAccessibility());
+      buttonDataComponent().contains("Open DialogFullScreen").click();
+      cy.checkAccessibility();
```

Comment:
shall we add something to be sure that Dialog is opened? Like the crossIcon is visible, or something similar?